CxScan/CxScanV20/services/configReader.ts

```diff
@@ -500,6 +501,7 @@ CxSAST thresholds enabled: ${config.sastConfig.vulnerabilityThreshold}`);
             if (config.sastConfig.vulnerabilityThreshold) {
                 this.log.info(`CxSAST fail build for new vulnerabilities enabled: ${config.sastConfig.failBuildForNewVulnerabilitiesEnabled}`);
                 this.log.info(`CxSAST fail build for the following severity or greater: ${config.sastConfig.failBuildForNewVulnerabilitiesSeverity}`);                    
+                this.log.info(`CxSAST critical threshold: ${formatOptionalNumber(config.sastConfig.criticalThreshold)}`);
```

Comment:
Check the condition when SAST is 9.6 or below it should not log Critical Threshold in the logs.

Reply:
While reading this we cannot check condition. As I discusses already with Fatima in UI CRITICAL dropdown and testbox will remain as it is for all SAST. Added information and warning message based on SAST version